test/test-applications/integrations/TestApplication.DomainNeutral/TestApplication.DomainNeutral.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
```

Comment:
Isn't the default a class library?

Reply:
Somehow there is still exe 🤷 
The point why it was removed, is with this option App.config defined redirections were not copied to output.exe.config.

No clue at all why.